ibtk/include/ibtk/LibMeshSystemVectors.h

```diff
@@ -57,7 +55,7 @@ namespace IBTK
  * ghost data computed by the System) and registered for projection (i.e., if
  * the mesh changes, these vectors will be automatically updated).
  *
- * This class stores information that depends on SAMRAI's parallel
+ * This class may store information that depends on SAMRAI's parallel
  * partitioning: LibMeshSystemVectors::reinit() should be called after
  * regridding to clear this data.
```

Comment:
This  comment is no longer true and should be removed; i.e., this class doesn't know anything about SAMRAI any more.